molecule/rbac-kafka-connect-replicator-kerberos-mtls-custom-ubuntu2004/prepare.yml

```diff
@@ -12,4 +12,4 @@
   import_playbook: confluent.platform.all
 
 - name: Create Topics and Data for Replication on Source Cluster
-  import_playbook: topic_data_creation.yml
```

Comment:
looks like this file was deleted

Reply:
We moved the file a level up and removed form each scenario directories

Reply:
yes this PR does not show that the file was moved, just deleted

Reply:
NVRM i see it now, thanks!